s3-private/s3.tf

```diff
@@ -1,15 +1,16 @@
-resource "aws_s3_bucket" "main" {
-  bucket = var.bucket_name
-
-  tags = {
-    Name        = var.bucket_name
-    Project     = var.project
-    Environment = var.environment
-  }
+module "s3" {
```

Comment:
we have to be careful when using this new module, since moving the "main" reference into a module will force a re-creation.

before: `module.storage-bucket.aws_s3_bucket.main`
after    : `module.storage-bucket.module.s3.aws_s3_bucket.main`